tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -256,9 +262,12 @@
             os_net_config_iface: {{ dataplane_os_net_config_iface | default ('nic1') }}
             os_net_config_set_route: {{ dataplane_os_net_config_set_route | default(true) | bool }}
             os_net_config_dns: {{ dataplane_os_net_config_dns | default("") }}
+
+            {%+ if compute_adoption|bool +%}
             service_net_map:
```

Comment:
those are no longer needed and being removed in edpm

Reply:
This would belong to a separate PR I think.